File: Android_Zone_Test/src/com/example/mylib_test/activity/touch/NestedScrollingActivity_hongParent.java
package com.example.mylib_test.activity.touch;

import android.support.v7.widget.LinearLayoutManager;
import android.support.v7.widget.RecyclerView;

import com.example.mylib_test.R;

import java.util.ArrayList;
import java.util.List;

import com.example.mylib_test.delegates.TextDelegates;
import com.example.mylib_test.delegates.TextType2Delegates;
import com.zone.adapter3.QuickRcvAdapter;
import com.zone.lib.base.activity.BaseActivity;

import butterknife.Bind;
import butterknife.ButterKnife;


public class NestedScrollingActivity_hongParent extends BaseActivity {
    @Bind(R.id.recyclerView)
    RecyclerView recyclerView;

    private List<String> mDatas = new ArrayList<String>();

    @Override
    public void setContentView() {
        setContentView(R.layout.activity_nested_parent_hong);
        ButterKnife.bind(this);
    }

    @Override
    public void findIDs() {

    }

    @Override
    public void initData() {
        for (int i = 0; i < 50; i++) {
            mDatas.add("Parent Demo -> " + i);
        }
        recyclerView.setLayoutManager(new LinearLayoutManager(this));

        new QuickRcvAdapter<String>(this, mDatas)
                .addViewHolder(new TextType2Delegates())
                .relatedList(recyclerView);
    }

    @Override
    public void setListener() {

    }

}

File: Android_Zone_Test/src/com/example/mylib_test/activity/touch/NestedScrollingActivity_hongParent.kt
package com.example.mylib_test.activity.touch


import com.example.mylib_test.R

import java.util.ArrayList
import com.example.mylib_test.adapter.delegates.TextType2Delegates

import androidx.recyclerview.widget.LinearLayoutManager
import com.zone.adapter3kt.QuickAdapter
import com.zone.lib.base.controller.activity.BaseFeatureActivity
import kotlinx.android.synthetic.main.activity_nested_parent_hong.*


class NestedScrollingActivity_hongParent : BaseFeatureActivity() {
    private val mDatas = ArrayList<String>()

    override fun setContentView() {
        setContentView(R.layout.activity_nested_parent_hong)
    }

    override fun initData() {
        for (i in 0..49) {
            mDatas.add("Parent Demo -> $i")
        }
        recyclerView.layoutManager = LinearLayoutManager(this)

        recyclerView.adapter = QuickAdapter<String>(this).apply {
            registerDelegate(TextType2Delegates())
            add(mDatas)
        }
    }

    override fun setListener() {
    }

}
